compiler/jsgen.nim

```
@@ -2282,6 +2282,9 @@ proc myClose(graph: ModuleGraph; b: PPassContext, n: PNode): PNode =
         else: AbsoluteFile(getCurrentDir() / m.config.outFile.string)
       else:
         changeFileExt(completeCFilePath(m.config, AbsoluteFile f), ext)
+    let (outDir, _, _) = splitFile(outfile)
+    if not outDir.isEmpty:
+      try: createDir(outDir) except: discard "couldn't create dir"
```

Let the exception bubble up, don't hide it.